src/app/[locale]/(pages)/restoreCode/page.tsx

```diff
@@ -39,24 +41,26 @@ const ReactivateCode = (): React.ReactElement => {
       <Grid sx={commonBackground} container>
         <Grid item xs={12} justifySelf={'center'} sx={{ pb: { xs: 0, md: '10px' } }}>
           <Introduction
-            title="Inserisci il codice di ripristino"
+            title={t('restore.insertrestorecode')}
+            // TODO: [TL] tags
             summary={
-              <>
-                Ha 9 cifre, ti è stato fornito in fase di blocco dell&apos;accesso a IO.
-                <br />
-                Non lo trovi?{' '}
-                <Link href="#" fontWeight={600}>
-                  Scopri cosa fare
-                </Link>
-              </>
+              t('restore.explanationrestorecode')
+              // <>
+              //   Ha 9 cifre, ti è stato fornito in fase di blocco dell&apos;accesso a IO.
+              //   <br />
```

Comment:
why there is this comment?

Reply:
I left the previous paragraphs commented as a memo to compare where and which are the tags we have to put in the text. Every one of these has a TODO to fix it when we'll have the correct texts with tags. 

Can you give me a new task so I can change the TODOs into FIXMEs with the new task number in order to pass this PR and come back later on these?